### PR TITLE
Create 9-2-Coaching-Expectations.md

### PR DESCRIPTION
**Pull request by ajc**
Alright, here's a first cut - what else do we want to say?

I completely guessed 9-2, of course - feel free to change it.

### CODE REVIEW

File: 9-2-Coaching-Expectations.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Some people will feel comfortable asking questions in our Slack channel, so we need a few people ready to answer them.
+
+## Table helpers
+
+Other participants might be more comfortable with in-person assistance, so we have a coach sitting at each table, ready to help.
+
+## Roamers
+
+A couple of people can wander the room, providing ad-hoc assistance where other helpers are already occupied.
```
Comment (millyrowboat) [inline 9-2-Coaching-Expectations.md]: Potentially mention that our coaches should encourage attendees to raise their hand when they need help; or let the coaches know to look for 'furrowed brows'. Our helpers for the Haskell workshop mentioned this and it put us at ease for asking or expecting help.

File: 9-2-Coaching-Expectations.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Latecomer helpers
+
+Latecomers might need a bit more help coming up to speed, so it’d help them to have a dedicated helper (perhaps not for the whole session, but at least while they get going).
+
+# What we expect of our coaches:
+
+- You’ve run through the course content beforehand, so you’re familiar with what people will be doing, and can explain what’s going on. While that might sound a bit scary, don’t panic! You don’t need to know everything, just enough to be able to point people in the right direction - you can always ask other coaches for help if you’re not sure about a particular question somebody has.
+- You’ll socialise with the participants during the event - we want them to feel welcome! If you see people standing alone during the breaks, have a chat to them and see how they’re going.
```
Comment (millyrowboat) [inline 9-2-Coaching-Expectations.md]: This is a really important point. We can't forbid coaches from hanging out with other coaches, but remind them that we're here for the attendees who probably feel awkward in a room of strangers.

File: 9-2-Coaching-Expectations.md (hunk 1 above)
Comment (ajc) [reply 9-2-Coaching-Expectations.md]: I've added a note about the second bit - re: the first comment, I feel like we want the speakers (either semi-regularly or as part of the rest of their intro for the day) to encourage people to raise their hand if they want help - where do we put that guidance?

**Comment by noemif** [comment]
👍 you captured all the roles we talked about really well Cos, thanks ! Nothing else I can think of right now. That should already make the next iteration way better :-)

**Pull request merged**